src/document/reducer.ts

```diff
@@ -83,6 +90,7 @@ function updateOperations<T extends Document>(
     document: T,
     action: Action | Operation,
     skip = 0,
+    reuseLastOperationIndex = false,
```

Comment:
Please add a `@param` comment for the `skip` and `reuseLastOperationIndex`

Reply:
fix: [e66bcca](https://github.com/powerhouse-inc/document-model/pull/89/commits/e66bcca058fbef5dbb2543fc155ac580437e1a61)